tests/integration.rs

```diff
@@ -5574,6 +5574,20 @@ mod tests {
             .unwrap_or(0);
         }
 
+        fn get_ram_gigs_windows(auth: &PasswordAuth) -> u8 {
```

Comment:
I would prefer the name `get_ram_size_windows()` and make this function return the size in bytes.

Reply:
When I change to compare bytes, the reported size is about ~1Mb smaller per slot. The initial size is reported that way, too - 2146267136 instead of 2147483648. It might be that some amount is used for servicing, or perhaps it's something powershell specific. I'll check more.

Thanks.

Reply:
An update on this - it looks like a constant `1216512`, not per slot. Adding further slots seems to have no effect, the diff doesn't change. As it looks like it's something system reserved and not related to this patch, i'm going to make a correction in teh assert for the time being. I'll follow up on what this amount could be about.

Thanks.

Reply:
Fixed, thanks.

Reply:
Hmm, the diff seems to be another number on the CI host - `1081344`. It took another round, i just made it check this constant doesn't go beyond 4mb, so fixed it that way.

Thanks